Splunk_TA_paloalto/default/eventtypes.conf

```diff
@@ -23,19 +23,30 @@ search = sourcetype=pan_traffic OR sourcetype=pan:traffic OR (sourcetype=pan:fir
 
 [pan_traffic_end]
 search = sourcetype=pan_traffic OR sourcetype=pan:traffic OR (sourcetype=pan:firewall_cloud AND LogType="TRAFFIC") AND log_subtype="end"
-#tags = network session end
+#tags = network communicate
 
 [pan_system]
 search = sourcetype=pan_system OR sourcetype=pan:system OR (sourcetype=pan:firewall_cloud AND LogType="SYSTEM")
-#tags = update status 
+
+[pan_system_auth]
+search = sourcetype=pan_system OR sourcetype=pan:system AND log_subtype="globalprotect" OR description="*Failed password*" NOT description="*client configuration released*" NOT description="*client configuration generated*"
+#tags = authentication default
+
+[pan_system_alert]
+search = sourcetype=pan_system OR sourcetype=pan:system AND log_subtype="url-filtering"
+#tags = alert
+
+[pan_system_change]
+search = sourcetype=pan_system OR sourcetype=pan:system description="*config cleared*" AND NOT (log_subtype IN ("routing", "ras", "vpn"))
+#tags = change
 
 [pan_threat]
-search = sourcetype=pan_threat OR sourcetype=pan:threat OR (sourcetype=pan:firewall_cloud AND LogType="THREAT") AND log_subtype != "url" log_subtype != "file" log_subtype != "wildfire" log_subtype != "data" 
+search = sourcetype=pan_threat OR sourcetype=pan:threat OR (sourcetype=pan:firewall_cloud AND LogType="THREAT") AND log_subtype != "url" log_subtype != "file"
```

Comment:
The description for this change says:

> "pan_threat - events with log_subtype=”wildfire” and log_subtype=”data” for pan:threat will not be mapped to Intrusion Detection DataModel anymore"

But this change does the opposite by including `wildfire` and `data` logs in the `pan_threat` eventtype which is tagged with IDS.

Reply:
Okay. I think we must have misread the query while writing the PR description.

Reply:
Gotcha, makes sense. I ended up merging this as I think it makes sense for these two subtypes to be part of pan_threat, especially now that we're tagging wildfire as IDS.